.goreleaser.yml

```diff
@@ -31,7 +31,9 @@ nfpm:
     - deb
   bindir: /usr/bin
   files:
-    "_release/cacophonator-management.service": "/etc/avahi/services/cacophonator-management.service"
+    "_release/cacophonator-management-avahi.service": "/etc/avahi/services/cacophonator-management.service"
+    "_release/cacophonator-management-systemd.service": "/etc/systemd/system/cacophonator-management.service"
+    "_release/management-interface.yaml": "/etc/cacophony/management-interface.yaml"
```

Comment:
This needs to be in `config_files` not `files` or else it'll get trampled when the package is upgraded.